src/client/components/Navigation/Header.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { signOut } from '../../firebase/auth';
+
+export default function Header({ isAuth = false }) {
+  if (isAuth) {
+    return (
+      <nav>
+        <Link to="/">Home</Link>
+        <Link to="/profile">Profile</Link>
+        <button type="button" onClick={signOut}>
+          Sign out
+        </button>
+      </nav>
```

Comment:
Same with the nav below

```suggestion
      <nav>
        <ul>
          <li>
            <Link to="/">Home</Link>
          </li>
          <li>
            <Link to="/profile">Profile</Link>
          </li>
          <li>
            <button type="button" onClick={signOut}>
              Sign out
            </button>
          </li>
      </nav>
```